### PR TITLE
update libvirt accept tcp connections and update golang

### PR DESCRIPTION
**Pull request by sallyom**
@praveenkumar I have updated the `openshift-gce-devel` with the following images and I confirm that the install succeeds with this update:
* openshift4-libvirt-source (packer-601e91fd-c5a9-ed1b-f617-77b3b6981877)
* openshift4-libvirt (packer-601e9c2d-0745-5bf5-e340-cb4142491203) 

I have _not_ updated  `openshift-gce-devel-ci`

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -14,8 +14,8 @@ sudo dnf versionlock add qemu-kvm-2.12.0-88.module+el8.1.0+5708+85d8e057.3
 sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++
 
 # Install golang
-curl -L https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -o go1.13.8.linux-amd64.tar.gz
-tar -xvf go1.13.8.linux-amd64.tar.gz
+curl -L https://dl.google.com/go/go1.15.8.linux-amd64.tar.gz -o go1.15.8.linux-amd64.tar.gz
+tar -xvf go1.15.8.linux-amd64.tar.gz
```
Comment (praveenkumar) [inline provision.sh:18]: Why can't we use `golang` from offical repo , it is `golang-1.14.12-1.module+el8.3.0+8784+380394dc.x86_64` as of now. Are we trying to build something on this VM which require latest golang?

Comment (sallyom) [reply provision.sh:18]: kube uses  go 1.15 as of v 1.19, so keeping in sync with rest of kube/openshift

Comment (cfergeau) [reply provision.sh:18]: Hrm, would be nice to get RHEL `go-toolset` updated to 1.15 if that's a requirement for openshift builds.

Comment (praveenkumar) [reply provision.sh:18]: @sallyom but we are not compiling the kube/openshift part on these VM for test right?

Comment (sallyom) [reply provision.sh:18]: no, we aren't so official repo golang should be fine.

**Comment by sallyom** [comment]
this was merged in another PR, closing

**Pull request closed without merging**